neighbors/src/components/mainLayout.css

```diff
@@ -0,0 +1,205 @@
+.home-container {
```

Comment:
Ten komentarz dotyczy ogólnie struktury plików CSS.
Generalnie łatwiej jest utrzymywać i kontrolować style, jeżeli są one porozmieszczane w różnych plikach (dzięki temu pliki są mniejsze). Ładnie jest to rozwiązane dla komponentu HomePanel, który ma własny plik css to styli.
Dobrze, że jest kilka plików css, ale można by popracować nad ich podziałem i rozmieszczeniem.